src/nbody_simulation_result.jl

```diff
@@ -72,15 +94,31 @@ function temperature(result::SimulationResult, time::Real)
     return temperature
 end
 
-function kinetic_energy(velocities, simulation::NBodySimulation)
-    masses = get_masses(simulation.system)
+function temperature(result::SimulationResult{<:WaterSPCFw}, time::Real)
+    kb = 1.38e-23
```

Comment:
maybe make this a global constant at higher accuracy? Using it in these units may be terrible for floating point though.

Reply:
MD simulations should be done in different units anyway. Using the OpenMM convention the Boltzmann constant should be in units of kJ / (mol * K), which results in a value of 0.00831446.

Reply:
I agree: this is not a proper place for this constant, and SI units of measurement are also not convenient.